k8s/create-kind.sh

```diff
@@ -22,6 +24,9 @@ kubectl apply -f deployments/m3-portal-backend-deployment.yaml
 kubectl apply -f deployments/m3-portal-frontend-deployment.yaml
 kubectl apply -f deployments/portal-proxy-deployment.yaml
 
+# create Prometheus configMap, this will be used by Prometheus Kubernetes Deployment
+kubectl apply -f deployments/prometheus-configmap.yaml
```

Comment:
this file is only used by developers, sounds like this config map is something that will be needed by production deployments, maybe embed this config map into `m3-deployment.yaml.example` to make sure people configure it properly

Reply:
makes sense, done!